Sofa/framework/Type/src/sofa/type/Mat.h

```diff
@@ -1252,24 +1252,6 @@ constexpr Mat<3,3,real> multTranspose(const Mat<3,3,real>& m1, const Mat<3,3,rea
 }
 
 #if !defined(SOFA_TYPE_MAT_CPP)
```

Comment:
You let the preproc test for a future usage ?

Reply:
Actually I was hopping to restore the extern template instantiations while keeping the inline. But I have no idea how to do that (without tuning the compiler options). So I guess I can remove it for now